src/CMakeLists.txt

```diff
@@ -121,7 +121,8 @@ endif()
 
 if(APPLE)
   list(APPEND OMVLL_LINK_OPT
-    #-undefined dynamic_lookup
+    -Wl,-flat_namespace -Wl,-undefined -Wl,suppress
```

Comment:
> -flat_namespace here is necessary due to using -Wl,-undefined,suppress, correct?

Yes

> What about if a symbol name (with multiple definitions) is encountered first in linked libraries rather than in our own library?

The only thing we care about is `_llvmGetPassPluginInfo`. Calng resolves it from the dylib manually with `dlsym()`. I don't think `-flat_namespace` affects that in any way.